src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -96,7 +132,7 @@ const useGroupSetting = ({navigation, route}) => {
         }
       );
     } else {
-      SimpleToast.show(message, SimpleToast.SHORT);
+      Linking.openSettings();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from showing a toast message to opening settings when the permission request fails is a significant behavior change. Ensure that this is the intended behavior and that it provides a better user experience.

```diff
- Linking.openSettings();
+ SimpleToast.show(message, SimpleToast.SHORT);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->